src/App.css

```diff
@@ -1,39 +1,56 @@
-.App {
```

Comment:
- [ ] You did a great job with the design. :+1: Kindly ensure to remove the boilerplate files created by CRA as suggested in the project requirements. 🥸️

![Screenshot from 2023-07-06 13-00-12](https://github.com/tjay1760/Math-Magician/assets/17243325/e6672ff6-4fcb-4cdd-80b4-5eae57ca03a9)

- The following files can be considered boilerplate files.

<img src="https://github.com/tjay1760/Math-Magician/assets/17243325/4a97ed8e-9dca-428d-a874-66d49be9f24a" width="300">